bundles/org.openhab.ui.dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Dashboard tiles
+
+OpenHAB dashboard is landing page for the user where all openHAB UI's can be found. Dashboard support also links to external services. Links can be added to dashboard by ```conf/services/dashboard.cfg``` configuration file.
+
+## Link Configuration
+
+| Parameter name  | Type    | Description                                                                             |
+|-----------------|---------|-----------------------------------------------------------------------------------------|
+| link-nameX      | String  | Name which is shown in the openHAB dashboard.                                           |
+| link-urlX       | String  | URL to external service.                                                                |
+| link-overlayX   | String  | Image overlay icon. Supported values are empty (no icon), "html5", "android" or "apple" |
```

Comment:
I wonder if in that case this feature at all makes sense - the overlay was used in the past, but as everything ended up using the HTML5 overlay, we removed it. So it is rather in there as some legacy, which we might not want to promote again through a new feature. Wdyt?